code/modules/holiday/holidays.dm

```diff
@@ -708,30 +708,6 @@
 /datum/holiday/programmers/getStationPrefix()
 	return pick("span>","DEBUG: ","null","/list","EVENT PREFIX NOT FOUND") //Portability
 
-// ISLAMIC
-
-/datum/holiday/islamic
-	name = "Islamic calendar code broken"
-
-/datum/holiday/islamic/shouldCelebrate(dd, mm, yyyy, ddd)
-	var/datum/foreign_calendar/islamic/cal = new(yyyy, mm, dd)
-	return ..(cal.dd, cal.mm, cal.yyyy, ddd)
-
-/datum/holiday/islamic/ramadan
-	name = "Start of Ramadan"
-	begin_month = 9
-	begin_day = 1
-	end_day = 3
-
-/datum/holiday/islamic/ramadan/getStationPrefix()
-	return pick("Haram","Halaal","Jihad","Muslim", "Al", "Mohammad", "Rashidun", "Umayyad", "Abbasid", "Abdul", "Fatimid", "Ayyubid", "Almohad", "Abu")
```

Comment:
I'm not very knowledgeable on these but some of them seems quite tasteless

Reply:
Haram: forbidden
Halaal: permissible
Jihad: struggle against enemies of Islam
Muslim: follower of Islam
Al: the
Mohammad: Praiseworthy; prophet
Rashidun, Umayyad, Abbasid: historic Islamic caliphates
Abdul: servant to god
Fatimid, Ayyubid, Almohad: historic Islamic dynasties
Abu: father (of)

The only two that I see as potentially tasteless are Jihad and (to a much lesser extent) Haram, but I'm no expert.

Reply:
The only thing I discovered reading this is that we are shameless to all people during their respective holiday except Christmas, which doesn't have funny station names.

Reply:
> The only thing I discovered reading this is that we are shameless to all people during their respective holiday except Christmas, which doesn't have funny station names.

this is blasphemy